pyproject.toml

```diff
@@ -23,7 +23,10 @@ maintainers = [
 ]
 readme = "README.rst"
 requires-python = "<4.0,>=3.9"
-dependencies = ["rpmautospec-core<1.0.0,>=0.1.4"]
+dependencies = [
+    "rpmautospec-core<1.0.0,>=0.1.4",
+    "norpm>=1.6",
```

Comment:
This should be an optional dependency (like rpm), and probably clamp the version to `<2`, too.

Reply:
Not a big deal! There just isn't any versioning scheme in norpm :angel: and, in general, I doubt these `< VERSION` restrictions are very practical (these conditions often just break RPM builds @ Fedora, even though the "updated dependency" stays compatible with the software).